src/aics-image-viewer/components/ColorPicker.jsx

```diff
@@ -58,45 +68,55 @@ class ColorPicker extends React.Component {
 
   render() {
     const width = this.props.width || 36;
-    const styles = reactCSS({
-      'default': {
-        color: {
-          width: `${width}px`,
-          height: '14px',
-          borderRadius: '2px',
-          background: `rgba(${ map(this.state.color, ele => (ele))})`
-        },
-        swatch: {
-          padding: '5px',
-          borderRadius: '1px',
-          boxShadow: '0 0 0 1px rgba(0,0,0,.1)',
-          display: 'inline-block',
-          cursor: 'pointer'
-        },
-        popover: {
-          position: 'absolute',
-          zIndex: '9999',
-        },
-        cover: {
-          position: 'fixed',
-          top: '0px',
-          right: '0px',
-          bottom: '0px',
-          left: '0px'
-        }
+    let styles = {
```

Comment:
should still be const?

Reply:
was adding a property to the style based on whether the color picker should open above or below swatch, which I agree was weird. just changed to be more consistent with other single-file components, with an all-caps `const STYLES` at the very bottom.

Reply:
Adding a property shouldn't have changed constness.   Would be surprised if you got some tool-based indication that you had to change it.  Unless you were reassigning the `styles` variable itself.

Reply:
I was conditionally adding either one property or another after declaration. You're right that with the way const works in JS it wouldn't have mattered, but in personal projects I tend to change the keyword to indicate whether any property of an element changes, and I hadn't reconsidered that habit until just now. Regardless, style-wise I feel better about the changes I just made.